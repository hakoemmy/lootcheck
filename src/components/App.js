import React, {Component} from 'react';

class App extends Component{
    render(){
        return (<div>
                 <h2>Lootcheck</h2>
              </div>);
    }
}

export default App;