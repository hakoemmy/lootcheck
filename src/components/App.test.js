import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
import '../setupTests';

describe('App component', () => {
   const app = shallow(<App />);

   it ('renders properly', () =>{
       expect(app).toMatchSnapshot();
   })
});