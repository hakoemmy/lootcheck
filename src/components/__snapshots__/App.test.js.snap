// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders properly 1`] = `ShallowWrapper {}`;
